frontend/src/app/organizations/organizations.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, OnInit } from '@angular/core';
+import { OrganizationsService } from './organizations.service';
+import { Organization } from '../organization/organization.model';
+import { OrganizationService } from '../organization/organization.service';
+import { Observable } from 'rxjs/Observable';
+
+@Component({
+  selector: 'volontulo-organizations',
+  templateUrl: './organizations.component.html',
+  styleUrls: ['./organizations.component.css'],
```

Comment:
Two things here:
 * I'm aware that this change was done before sass was introduced, but since #808 we use sass instead of css
 * We decided to not store empty style files, so remove the file and this attribute

Reply:
styleUrls and css files was deleted